doc/source/development/code_style.rst

```diff
@@ -159,3 +159,25 @@ For example:
 
     # wrong
     from common import test_base
+
+Testing
+=======
+
+Reading from a url
+------------------
+
+.. code-block:: python
```

Comment:
can you add \*\*Good:\*\* above this for consistency. (Personally, I prefer yes/no over good/bad, but let's leave that for another day)

same for next section